frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlDataHelpers.js

```diff
@@ -500,6 +507,22 @@ export const isHidden_lt_OCP48 = (control, controlData) => {
     return true
 }
 
+export const isHidden_gt_OCP46 = (control, controlData) => {
+    const singleNodeFeatureFlag = controlData.find(({ id }) => id === 'singleNodeFeatureFlag')
+    const imageSet = controlData.find(({ id }) => id === 'imageSet')
+    //NOTE: We will need to adjust this in the future for new OCP versions!
```

Comment:
This comment is not really accurate... the function checks for anything 4.6 (or older) but 4.6 is the oldest we've ever supported deploying, so we will not need to add new versions here.